pep-8104.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 8104
+Title: 2023 Term steering council election
+Version: $Revision$
+Last-Modified: $Date$
```

Comment:
```suggestion
```

No need for these long-obsolete legacy headers that are no longer part of the PEP template

Reply:
Already addressed in Adam's simultaneous review